### ISSUE TITLE
Feature Request:  Add Environment details to crash issues - reported by @Santhosh-Sellavel 

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#10518]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Problem: 
From the crash issue description we are unable to distinguish which environment it occurred.

## Solution: 
If possible we should add environment details to the issue description for crash issues, if possible let's update environment-wise stats there.

## Workaround:
unknown 

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 
**Reproducible in staging?:** 
**Reproducible in production?:** 
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
<img width="889" alt="Screenshot 2022-08-08 at 11 19 01 PM" src="https://user-images.githubusercontent.com/43995119/186325860-b03d5414-1e35-454d-aff8-a7e827d1588d.png">

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @Santhosh-Sellavel 
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1659981562906149

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by flodnv** [comment]
@Santhosh-Sellavel what do you mean by environment? cc @AndrewGable

**Comment by AndrewGable** [comment]
I believe it means Dev/Staging/Prod

**Comment by Santhosh-Sellavel** [comment]
> I believe it means Dev/Staging/Prod
Yes 

@flodnv no one is assigned here

**Comment by flodnv** [comment]
@AndrewGable I don't think this can be External...?

**Comment by AndrewGable** [comment]
I agree, seems internal to me too.

**Comment by Santhosh-Sellavel** [comment]
@AndrewGable there is no one assigned here

**Comment by Santhosh-Sellavel** [comment]
@flodnv or @AndrewGable No one is assigned here

**Comment by AndrewGable** [comment]
What is your point @Santhosh-Sellavel?

**Comment by Santhosh-Sellavel** [comment]
Can you rephrase @AndrewGable?

**Comment by AndrewGable** [comment]
I'm wondering why you are commenting saying that it's unassigned? If it's an internal issue, an Expensify engineer will pick it up and work on it when they have time. Otherwise, it will just sit unassigned, so there is no need to comment on it.

**Comment by Santhosh-Sellavel** [comment]
The reason I was bumping is that I think this could be useful information. Recently on this [issue](https://github.com/Expensify/App/issues/11288) it seemed that it occurs during development. this [one](https://github.com/Expensify/App/issues/11103) also.

**Comment by AndrewGable** [comment]
Got it! That is valuable information for sure, thank you! 🙏

**Comment by johncschuster** [comment]
I'm going to close this per [this conversation](https://expensify.slack.com/archives/C01GTK53T8Q/p1666117401531069?thread_ts=1666110643.035949&cid=C01GTK53T8Q).
